cli/jobs/automl-standalone-jobs/cli-automl-image-object-detection-task-fridge-items/cli-automl-image-object-detection-task-fridge-items.yml

```diff
@@ -39,8 +40,10 @@ sweep:
     slack_factor: 0.2
     delay_evaluation: 6
 
+# <search_space_settings>
```

Comment:
Remove this. I added the reasoning for this in a comment on your docs PR. Please refer to that for more details.